dvc/repo/run.py

```diff
@@ -53,13 +53,12 @@ def run(
 
     self.check_dag(self.stages() + [stage])
 
-    self.files_to_git_add = []
```

Comment:
And here.

Reply:
As discussed, wrapping with scm_context.